src/policy/mallocspace/metadata.rs

```diff
@@ -72,19 +148,52 @@ pub fn is_alloced(object: ObjectReference) -> bool {
     is_alloced_object(object.to_address())
 }
 
+#[allow(unused)]
 pub fn is_alloced_object(address: Address) -> bool {
     side_metadata::load_atomic(ALLOC_SIDE_METADATA_SPEC, address, Ordering::SeqCst) == 1
 }
 
-pub fn is_marked<VM: VMBinding>(object: ObjectReference) -> bool {
+#[allow(unused)]
```

Comment:
Why do we have so many unused functions?

Reply:
We don't -- I just put it on all of those functions out of habit. I'll fix it up.